cfme/common/provider_views.py

```diff
@@ -325,6 +325,18 @@ def is_displayed(self):
                 self.entities.title.text == 'Middleware Providers')
 
 
+class PhysicalProvidersView(ProvidersView):
+    """
+     represents Main view displaying all Infra providers
+    """
+    @property
+    def is_displayed(self):
+        return (super(PhysicalProvidersView, self).is_displayed and
```

Comment:
I was gonna eek here on the super, but since we have a single inheritance, I can close one eye :)